src/Symfony/Component/Validator/Constraints/UrlValidator.php

```diff
@@ -90,6 +90,8 @@ public function validate($value, Constraint $constraint)
                 throw new InvalidOptionsException(sprintf('Invalid value for option "checkDNS" in constraint %s', get_class($constraint)), array('checkDNS'));
             }
 
+            @trigger_error(sprintf('The "checkDNS" option in "%s" is deprecated since 4.1 and will be removed in 5.0.', Url::class), E_USER_DEPRECATED);
```

Comment:
since *version* 4.1
I'm wondering: should we add a few words why? like eg
The "checkDNS" option in "%s" is deprecated since 4.1 and will be removed in 5.0. It's too fragile to be relied upon?

Reply:
the notice should be before the above "if", isn't it?

Reply:
Not sure if this is too fragile, but the false positive rate is so high that it's not reliable. I think reliable is perhaps better.